db/migrate/20160720194304_add_user_id_to_projects.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToProjects < ActiveRecord::Migration[5.0]
+  def change
+    add_column :projects, :user_id, :integer
```

Comment:
I think you missed foreign key reference here. It only adds integer column `user_id` without related database constraint. Take a look http://stackoverflow.com/a/31882059